### PR TITLE
Update NZ FIR Boundaries

### PR DESCRIPTION
**Pull request by vatpacais**

### CODE REVIEW

File: FIRBoundaries.dat [hunk 1]
```diff
@@ -13732,41 +13721,23 @@ NZOH|0|0|31|-48.150000|165.300000|-33.672351|180.000000|-40.000000|176.500000
...
--41.711111|173.521361
--41.507500|173.261111
--41.204806|171.992556
--41.073056|170.933611
+-38.843280|175.250897
+-39.334910|174.986144
+-40.5853802|174.986144
+-40.5853802|174.986144
+-40.7038501|175.4030622
+-40.7038501|175.4030622
```
Comment (NelisV) [inline FIRBoundaries.dat:13729]:
```suggestion
```
duplicate vertex

File: FIRBoundaries.dat [hunk 2]
```diff
@@ -13732,41 +13721,23 @@ NZOH|0|0|31|-48.150000|165.300000|-33.672351|180.000000|-40.000000|176.500000
...
--41.853278|174.682667
--41.703556|174.166806
--41.711111|173.521361
--41.507500|173.261111
--41.204806|171.992556
--41.073056|170.933611
+-38.843280|175.250897
+-39.334910|174.986144
+-40.5853802|174.986144
+-40.5853802|174.986144
```
Comment (NelisV) [inline FIRBoundaries.dat:13727]:
```suggestion
```
duplicate vertex

**Review by NelisV — CHANGES_REQUESTED** [review]
2 Duplicate vertexes, aside from those looks good 👌.

**Comment by Adrian2k** [comment]
Did you remeber to lower the point count by 1? If not VATSpy will throw an error message at you.

**Comment by vatpacais** [comment]
I'll resubmit_

**Review by Adrian2k — APPROVED** [review]

**Comment by NelisV** [comment]
No need to worry about those errors mate :) Looks good now. 
image

**Review by NelisV — APPROVED** [review]

**Pull request merged**